### PR TITLE
(fix)03-3338 :Refactor extraction of previous value to the section component.

### PR DESCRIPTION
**Pull request by gitcliff**
## Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) label. See existing PR titles for inspiration.
- [ ] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://zeroheight.com/23a080e38/p/880723-introduction).
- [ ] My work includes tests or is validated by existing tests.

## Summary
This PR moves the extraction of the value from the previous value object to happen from the section component, such that only the value is passed down to the input control and not the entire object
## Related Issue
https://openmrs.atlassian.net/browse/O3-3338

### CODE REVIEW

**Review by arodidev — APPROVED** [review]
LGTM, thanks @gitcliff

**Review by CynthiaKamau — APPROVED** [review]

File: src/components/inputs/content-switcher/content-switcher.component.tsx [hunk 1]
```diff
@@ -20,7 +20,7 @@ const ContentSwitcher: React.FC<FormFieldProps> = ({ question, onChange, handler
 
   useEffect(() => {
     if (!isEmpty(previousValue)) {
-      const { value } = previousValue;
+      const value = previousValue;
```
Comment (samuelmale) [inline src/components/inputs/content-switcher/content-switcher.component.tsx:23]: Do we need the `value` state?

Comment (gitcliff) [reply src/components/inputs/content-switcher/content-switcher.component.tsx:23]: thanks @samuelmale ,,,going to raise a fix for this

Comment (gitcliff) [reply src/components/inputs/content-switcher/content-switcher.component.tsx:23]: > Do we need the `value` state?

@samuelmale added the[ fix ](https://github.com/openmrs/openmrs-form-engine-lib/pull/306)

**Pull request merged**